modules/core/arrow-instances-data/src/main/kotlin/arrow/instances/kleisli.kt

```diff
@@ -33,69 +51,54 @@ interface KleisliContravariant<F, D> : Contravariant<Conested<Kind<ForKleisli, F
 @extension
 interface KleisliApplicativeInstance<F, D> : Applicative<KleisliPartialOf<F, D>>, KleisliFunctorInstance<F, D> {
 
-  fun AF(): Applicative<F>
+  override fun FFF(): Applicative<F>
```

Comment:
Can you do this? We've a few of these through the codebase, and we reasigned them instead, like:

override fun FFF() = AFF()

fun AFF(): Applicative<F>

Reply:
Added #1104 to work on that separately. Ideally we'd have the complete instance hierarchy for all the data types following this approach which reduces the amount of boilerplate quite a bit.